pages/ownerInformationPage.ts

```diff
@@ -79,9 +79,31 @@ export class OwnerInformationPage extends HelperBase{
     }
 
     async validatePetType(petName: string, petType: string){
-        const rosyPetType = this.page.getByText(petName).locator('..').locator('dd').last()
-        await expect(rosyPetType).toHaveText(petType)
+        const recievedPetType = this.page.getByText(petName).locator('..').locator('dd').last()
+        await expect(recievedPetType).toHaveText(petType)
 
     }
 
+    async validateOwnerInformationFromJSON(ownerObject: {}){
+
+        const ownerTable = this.page.getByRole('table').first()
+        await expect(ownerTable.locator('td').first()).toHaveText(`${ownerObject['firstName']} ${ownerObject['lastName']}`)
+        await expect(ownerTable.locator('td').nth(1)).toHaveText(ownerObject['address'])
+        await expect(ownerTable.locator('td').nth(2)).toHaveText(ownerObject['city'])
+        await expect(ownerTable.locator('td').last()).toHaveText(ownerObject['telephone'])
+    }
+
+    async validatePetNamesFromJSON(petObjectList: {}[]){
+
+        for(let petObject of petObjectList){
+            await expect(this.page.getByRole('table').nth(1)).toContainText(petObject['name'])
```

Comment:
Same here... the confusion is which `patObjectList` should be used? Make it simple. Also, make locator nicer removing indexes.

Reply:
I tweaked this quite a bit.  Let's see if it is up to snuff.

Reply:
But I am struggling with the table indices.  Maybe my approach is off.  Ok, I need to take this one to slack.  I made a commit all confident, and then I saw that glaring locator, lol